guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -2,7 +2,21 @@
 = Katello {KatelloVersion} Release Notes
 
 [id="katello-headline-features"]
-== Headline Features
+For the full list of changes, see the https://github.com/Katello/katello/blob/KATELLO-{KatelloVersion}/CHANGELOG.md[Changelog].
+
+* Simple Content Access is now the default for newly created organizations.
+- Simple Content Access no longer requires a manifest to be imported.
+- A Simple Content Access checkbox has been added to the Organization Create screen.
+- SCA status is no longer coupled to Candlepin's upstream consumer. Therefore, you can no longer toggle SCA from the Manage Manifest screen in Subscriptions. To toggle SCA, use the Organization Create or Edit screens.
+* On the content view versions page, you can now compare the content of any two versions.
+* Simplified Alternate Content Sources (ACSs) are now available for testing.
+- ACSs are alternative locations to download yum and file content from during syncing.  Enable ACSs in the UI via the "Show Experimental Labs" setting.
+- You can now edit ACSs in the UI.
+* Work on the new host details page has progressed:
+- 'System purpose' card is now available on the new host details / Overview tab
+- Tracer has been added to 'System properties' card on host details page.
+- 'Hardware properties' card has been added to the host details.
+* Over 30 bug fixes.
```

Comment:
Long term I'd like to align release note styles between Foreman and Katello. See https://docs.theforeman.org/3.3/Release_notes/index-foreman-el.html and https://docs.theforeman.org/3.1/Release_notes/index-foreman-el.html for some more inspiration. Interestingly enough, 3.2 is missing headline features.